src/components/widgets/TreeView/TreeViewLeaf.js

```diff
@@ -13,6 +13,7 @@ const TreeViewLeaf = ({
   title,
   admins,
   organizational,
+  archived,
```

Comment:
Actually, I am not happy about this. TreeView was supposed to be an universal widget. My intention is to remove all group-specific dependencies from TreeView once an opportunity for refactoring presents itself.